### ISSUE TITLE
Proposal for improvement: change icon

### ISSUE THREAD

**Issue by vpiasta** [ShadyThGod/altus#41]
Excellent app, especially the fact that the taskbar icon sets a red flag when new messages arrive.
Unfortunately the original Altus icon is hardly visible on a dark taskbar, so it would be a good feature to be able to change it. You either might create a whatsapp-like icon or allow users to replace the original icon with one of their choice. Changing the icon via the properties of the app does not work because then the red mark for incoming messages is not available anymore. I presume that you are using two different icons, one with the red mark and another without it. Allowing the user to replace both would do the trick.
Thanks

**Comment by ShadyThGod** [comment]
It actually does not use two different icons, but it's dynamically generated using Electron APIs. But I guess it doesn't work when the icon is changed through properties because Electron uses a set icon file to draw the red mark upon rather than using the icon set by the shell. Although, that is something I probably won't be able to change. Same with letting the user change the icon; I don't know how I'll be able to properly implement it, if that's even possible.

**Comment by vpiasta** [comment]
Hello Aman,

Thank you for coming back to my proposal. I can now confirm that your wrapper Works exactly as I have proposed, although one needs to use a little trick. 
Initially I had tried to change the icon of the app itself. This does not work. 
But then I created a link to it and changed the icon of the link to that of WhatsApp. Then I clicked the link with the new icon and fixed it on the taskbar. When a new message arrives for the WhatsApp account that is wrapped by Altus, the red spot is shown on the WhatsApp icon, exactly as I hoped to do. When the message is read and the flag is cleared, the red spot disappears also.
Obviously this would work with any kind of icon, so there does not need to be any confusión when there are 2 identical WhatsApp icons visible on the taskbar.
So you don’t need to invent anything, you just might pass the information how it can be done.

Thanks for your great work!
Volker

Enviado desde Correo para Windows 10

De: Aman Harwara
Enviado: sábado, 30 de noviembre de 2019 6:49
Para: ShadyThGod/altus
CC: vpiasta; Author
Asunto: Re: [ShadyThGod/altus] Proposal for improvement: change icon (#41)

It actually does not use two different icons, but it's dynamically generated using Electron APIs. But I guess it doesn't work when the icon is changed through properties because Electron uses a set icon file to draw the red mark upon rather than using the icon set by the shell. Although, that is something I probably won't be able to change. Same with letting the user change the icon; I don't know how I'll be able to properly implement it, if that's even possible.
